### PR TITLE
Add feedbacks to workspace usage API

### PR DESCRIPTION
**Pull request by albandum**
## Description

- Add feedbacks to workspace usage API
- Add by date range

## Deploy Plan
https://github.com/dust-tt/dust/pull/8879 needs to be merged first

### CODE REVIEW

**Review by JulesBelveze — APPROVED** [review]
LGTM, left a general nit 😃

File: front/lib/workspace_usage.ts [hunk 1]
```diff
@@ -71,6 +72,16 @@ interface AgentUsageQueryResult {
   lastConfiguration: Date;
 }
 
+interface FeedbackQueryResult {
+  created_at: Date;
+  userName: string;
+  userEmail: string;
+  agentConfigurationId: string;
+  agentConfigurationVersion: number;
+  thumb: string;
```
[the remark this answers is not in the document]
Comment (albandum) [reply front/lib/workspace_usage.ts:81]: Actually no - we don't have any FKs on these

[the remark this answers is not in the document]
Comment (JulesBelveze) [reply front/lib/workspace_usage.ts:81]: `thumbDirection` is an enum isn't it?

[the remark this answers is not in the document]
Comment (albandum) [reply front/lib/workspace_usage.ts]: indeed

**Pull request merged**